decidim-api/lib/tasks/decidim_api_tasks.rake

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
```

Comment:
We have merged this same file as `decidim-api/lib/tasks/decidim_api_docs.rake` in #8631 

Also the two images (decidim-api/app/packs/images/navbar.png and decidim-api/app/packs/images/menu.png) doesn't seem to be used for anything. For what I've seen pretty much this whole commit (b7973527fa11ea35645ef90130b2d23f13c3dcac) should be out as it was a test related with that PR or something like that

Reply:
just pushed the commit to remove 3 files ( this rake task and 2 pngs)